README.md

```diff
@@ -431,3 +431,65 @@ message.send()
 ```
 
 **[⬆ back to top](#table-of-contents)**
+
+### Functions should only be one level of abstraction
+
+When you have more than one level of abstraction, your function is usually doing too 
```

Comment:
I feel this is too similar to section `Functions should do one thing`

Reply:
I agree. Adapted from https://github.com/jupeter/clean-code-php#functions-should-only-be-one-level-of-abstraction -- it does appear to be a rephrasing of `Functions should do only one thing`.